### ISSUE TITLE
How to handle resources that have no "delete" method?

### ISSUE THREAD

**Issue by zikalino** [Azure/magic-module-specs#74]
I have enocountered several APIs that have no delete method, so I can't provide it in api.yaml
Not providing "delete" causes MM to fail on such yaml file.
How this should be handled?

**Comment by JunyiYi** [comment]
I don't think we need to generate any resources without `delete` because Terraform cannot handle that. Instead, I suggest we just implement it in Ansible manually.
